widgets/src/main/java/org/pentaho/gwt/widgets/client/buttons/ThemeableImageButton.java

```diff
@@ -158,7 +205,7 @@ public void onBrowserEvent( Event event ) {
     super.onBrowserEvent( event );
 
     // This is required to prevent a drag & drop of the Image in the edit text.
-    DOM.eventPreventDefault( event );
+    // DOM.eventPreventDefault( event );
```

Comment:
Commented out code should be removed unless there's a good reason to keep it (e.g. maintenance/documentation purposes).

Reply:
Sure. the constructor has been removed.